CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 [Will McGinty](https://github.com/willmcginty)
 [#76](https://github.com/BottleRocketStudios/iOS-UtiliKit/pull/76)
 
+* Added Swift Package Manager support.
```

Comment:
This should be listed under the `##### Enhancements` section above.

Reply:
Fixed.